README.md

```diff
@@ -1,24 +1,92 @@
-# gender-bias-web
-
-## Project setup
+# Reading For Gender Bias - FrontEnd (GEL team)
+​
+## Table of contents 
+​
+* [Introduction](#Introduction)
+* [Technologies](#Technologies)
+* [Installation and Usage](#Installation)
+* [Resources](#Resources)
+* [Features](#Features)
+* [Limitations](#Limitations)
+* [Credits](#Credits)
+​
+## Introduction
+​
+This project is a user-interface web application that employs *Reading for Gender Bias* back-end, [more information here](https://github.com/glam-lab/gender-bias) to create an accessible, text-analysis tool that directs recommenders and evaluators to avoid implicit gender biases in their writings. This process is done through flagging gendered words, phrases, and grindstone adjectives, pointing out shortfalls, and showing a checklist of requirements in a recommendation letter.
+​
+This project is currently a prototype that is under modification process to be turned into a usable product. Current modifications are monitored through Whitman College Computer Science Summer Research team.
+​
+## Technologies
+​
+* Javascript
+* Vue Framework
+* Puppeteer, mocha, chai for testing
+* HTML
+* CSS
+* yarn/npm as package managers
+​
+## Installation and Usage
+​
+### Backend installation process
+​
 ```
-yarn install
+git clone https://github.com/gender-bias/gender-bias
+cd gender-bias
+pip3 install -e .
 ```
-
-### Compiles and hot-reloads for development
+> NOTE: The last line in the above snippet installs this library in "editable" mode, which is probably fine while the library is in a state of flux.
+​
+### Package Managers to run the project
+​
+To set up the project, install a package manager such as yarn or npm.
 ```
-yarn serve
+yarn install
 ```
-
-### Compiles and minifies for production
+Or 
+```
+npm install
+​
+``` 
+### Running the front-end 
+​
+Once installed, add install Vue, and use yarn to open the server locally. Go to the right directory, and start the server. 
```

Comment:
I don't think this sentence adds anything to the commands below, so I would delete it. 

Software developers are impatient people. Concision and simplicity are great virtues in technical writing.